### DISCUSSION TITLE
unstable_revalidate not working, ways to troubleshoot?

### DISCUSSION THREAD

**Discussion by josh-unwin** [vercel/next.js#15157] [Help] [answered]
Hi there,

I'm trying out Incremental Static Regeneration (unstable_revalidate) at the moment. I have a slider banner on my index page which is populated with images/data from getStaticProps. I've added the `unstable_validate` flag to my returned props object but it doesn't seem to be having any effect either on Vercel or locally when doing `yarn build + yarn start`.

When I add a new image for the slider in my CMS it's not changing on the site (I've left it for ages to test now).

Reporting as a potential bug, or maybe I'm missing something. If there's any way to troubleshoot let me know.

index.tsx:
```
export async function getStaticProps() {
  const apolloClient = initializeApollo();

  await apolloClient.query({
    query: HOME_BANNER_QUERY,
  })

  return {
    props: {
      bannerData: apolloClient.readQuery({query: HOME_BANNER_QUERY}),
      unstable_revalidate: 1
    },
  }
}
```

**Comment by givi-phirtskhalava** [comment, marked as answer]
I believe you need to move it out of the props:

```
return {
 props: {...},
 unstable_revalidate: 1
}
```

**Reply by josh-unwin** [reply]
Oh. My. God. I'm an idiot. Thank you. I looked at that example about three times and was sure it was being put in the props. -_-

Working.
